nixos/modules/services/hardware/amdgpu.nix

```diff
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.hardware.amdgpu;
+in {
+  options.hardware.amdgpu = {
+    legacySupport.enable = lib.mkEnableOption ''
+      support for old AMD graphics cards, this uses `amdgpu`
+      instead of `radeonsi` driver for Southern Islands (Radeon HD 7000)
+      series and Sea Islands (Radeon HD 8000) series cards.
+      Note: this removes support for analog video output graphics cards,
+      which is only available in `radeonsi` driver
```

Comment:
```suggestion
      using `amdgpu` kernel driver instead of `radeon` for Southern Islands
      (Radeon HD 7000) series and Sea Islands (Radeon HD 8000)
      series cards. Note: this removes support for analog video outputs,
      which is only available in the `radeon` driver
```